build/dependencies.props

```diff
@@ -10,7 +10,7 @@
     <MicrosoftAspNetCoreApp31PackageVersion>3.1.3</MicrosoftAspNetCoreApp31PackageVersion>
     <MicrosoftBuildLocatorPackageVersion>1.2.2</MicrosoftBuildLocatorPackageVersion>
     <MicrosoftBuildPackageVersion>16.0.461</MicrosoftBuildPackageVersion>
-    <MicrosoftCodeAnalysisNetAnalyzersPackageVersion>6.0.0</MicrosoftCodeAnalysisNetAnalyzersPackageVersion>
+    <MicrosoftCodeAnalysisNetAnalyzersPackageVersion>7.0.0-preview1.22168.1</MicrosoftCodeAnalysisNetAnalyzersPackageVersion>
```

Comment:
Why did we need to bump analyzer dependencies here?

Reply:
Warnings are generated on build if an older version of this package is used with a newer SDK. Our build complained that 6.x was used with .NET 7.